hazelcast-build-utils/src/main/java/com/hazelcast/buildutils/FinalRemovalAgent.java

```diff
@@ -89,10 +93,15 @@ public static void attachFinalRemovalAgent() {
         if (FINAL_METHODS.containsKey(typeName)) {
             for (String methodName : FINAL_METHODS.get(typeName)) {
                 builder = builder.visit(new ModifierAdjustment()
-                                .withMethodModifiers(named(methodName), MethodManifestation.PLAIN)
+                        .withMethodModifiers(named(methodName), MethodManifestation.PLAIN)
                 );
             }
         }
         return builder;
     }
+
+    public static <T> Set<T> setOf(T... items) {
+        List<T> list = Arrays.asList(items);
+        return new HashSet<T>(list);
```

Comment:
maybe it is better to throw exception if items contains duplicate elements?

Reply:
done